class/50-host-classes

```diff
@@ -9,6 +9,9 @@ fi
 # START HOSTNAME TO CLASS MAPPING #
 ###################################
 case $HOSTNAME in
+    mariajail*)
```

Comment:
Oh no! it looks like mariajail is defined lower in this file already! you can revert this change, and then i can merge the PR. :) :+1: